tools/xcodeproj_shims/build-wrapper.sh

```diff
@@ -22,6 +22,9 @@ fi
 if [ -n "${TARGET_DEVICE_IDENTIFIER:-}" ] && [ "$PLATFORM_NAME" = "iphoneos" ]; then
     echo "Builds with --ios_multi_cpus=arm64 since the target is an iOS device."
     BAZEL_BUILD_OPTIONS+=("--ios_multi_cpus=arm64")
+elif [ $FORCE_X86_SIM -gt 0 ] && [ "$PLATFORM_NAME" = "iphonesimulator" ] && [[ $(sysctl -n machdep.cpu.brand_string) =~ "Apple" ]]; then
+    echo "Builds with --ios_multi_cpus=x86_64 since the target is sim, host architecture is apple silicon, and xcodeproj attribute force_x86_sim_on_apple_silicon is True"
```

Comment:
This can also be simplified to be something like:
Using x86_64 arch for simulator build because of the flag